src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -0,0 +1,194 @@
+    });
+  }
+});
+
+Cypress.Commands.add('fillTextAreaWebComponent', (field, value) => {
+  if (typeof value !== 'undefined') {
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-textarea[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('textarea')
+      .as('currentElement')
+      .type(value);
+
+    cy.get('@currentElement').then(el => {
+      if (el.val()) cy.get(el).should('have.value', value);
+    });
+  }
+});
+
+Cypress.Commands.add('selectRadioWebComponent', (field, value) => {
+  if (typeof value !== 'undefined') {
+    cy.get(`va-radio-option[name="${field}"][value="${value}"]`)
+      .as('currentElement')
+      .click();
+    cy.get('@currentElement').should('have.attr', 'checked');
+  }
+});
+
+Cypress.Commands.add('selectYesNoWebComponent', (field, value) => {
+  const selection = value ? 'Y' : 'N';
+  cy.selectRadioWebComponent(field, selection);
+});
+
+Cypress.Commands.add('selectDropdownWebComponent', (field, value) => {
+  if (typeof value !== 'undefined') {
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-select[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('select')
+      .as('currentElement')
+      .select(value, FORCE_OPTION);
+    cy.get('@currentElement').should('have.value', value);
+  }
+});
+
+Cypress.Commands.add('selectCheckboxWebComponent', (field, isChecked) => {
+  const element =
+    typeof field === 'string'
+      ? cy.get(`va-checkbox[name="${field}"]`)
+      : cy.wrap(field);
+
+  element
+    .shadow()
+    .find('input')
+    .as('currentElement');
+
+  if (isChecked) {
+    cy.get('@currentElement').check(FORCE_OPTION);
+    cy.get('@currentElement').should('be.checked');
+  } else {
+    cy.get('@currentElement').uncheck(FORCE_OPTION);
+    cy.get('@currentElement').should('not.be.checked');
+  }
+});
+
+Cypress.Commands.add('fillMemorableDateWebComponent', (field, value) => {
+  // eslint-disable-next-line cypress/no-assigning-return-values
+  const element = cy.get(`va-memorable-date[name="${field}"]`);
+
+  const [year, month, day] = value.split('-').map(
+    dateComponent =>
+      // eslint-disable-next-line no-restricted-globals
+      isFinite(dateComponent)
+        ? parseInt(dateComponent, 10).toString()
+        : dateComponent,
+  );
+
+  if (navigator.userAgent.includes('Chrome')) {
+    // There is a bug only on Chromium based browsers where
+    // VaMemorableDate text input fields will think they are
+    // disabled if you blur focus of the window while the test
+    // is running. realPress and realType solve this issue,
+    // but these are only available for Chromium based browsers.
+    // See cypress-real-events npmjs for more info.
+    element
+      .shadow()
+      .find('va-select.usa-form-group--month-select')
+      .shadow()
+      .find('select')
+      .as('monthSelect')
```

Comment:
There are instances of `va-memorable-date` being used without a month select - it's included it in the [command form helpers](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/testing/e2e/cypress/support/commands/formHelpers.js#L89)

Reply:
resolved, good call